scripts/us_eia/opendata/process/nuclear.py

```diff
@@ -0,0 +1,145 @@
+# Copyright 2021 Google LLC
```

Comment:
This shouldn't be part of this PR?

Reply:
merged in changes from other pr. i miss diffbase :-/